<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mahasiswa App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>720</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #800080;
color: white;
font-weight: bold;
font-size: 16pt;
</string>
    </property>
    <property name="text">
     <string>MAHASISWA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelNpm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>NPM</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNpm">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>550</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelNamaLengkap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>NAMA LENGKAP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNamaLengkap">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>550</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelNamaPanggilan">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>NAMA PANGGILAN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNamaPanggilan">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>550</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTelepon">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>TELEPON</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTelepon">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>550</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelEmail">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>EMAIL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEmail">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>550</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelKelas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>KELAS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKelas">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>550</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelMatakuliah">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>MATAKULIAH</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMatakuliah">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>550</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelLokasiKampus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>LOKASI KAMPUS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLokasiKampus">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>550</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoad">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #800080;
color: white;
border-radius: 5px;
padding: 5px;
</string>
    </property>
    <property name="text">
     <string>TAMBAH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSimpan">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #800080;
color: white;
border-radius: 5px;
padding: 5px;
</string>
    </property>
    <property name="text">
     <string>UBAH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #800080;
color: white;
border-radius: 5px;
padding: 5px;
</string>
    </property>
    <property name="text">
     <string>HAPUS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBatal">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #800080;
color: white;
border-radius: 5px;
padding: 5px;
</string>
    </property>
    <property name="text">
     <string>BATAL</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>700</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
